package/Dockerfile

```diff
@@ -19,5 +20,5 @@ VOLUME /var/lib/cni
 VOLUME /var/log
 ENV PATH="$PATH:/bin/aux"
 ENV CRI_CONFIG_FILE="/var/lib/rancher/k3s/agent/etc/crictl.yaml"
-ENTRYPOINT ["/bin/k3s"]
+ENTRYPOINT ["/bin/entrypoint.sh"]
```

Comment:
The logic should be coded in k3s agent, not in a shell script IMHO

Reply:
sounds like we're in agreement that this should go into the k3s agent code alongside the other cgroup handling stuff.